tests/test_odl_thorlabs_kbd101.py

```diff
@@ -0,0 +1,8 @@
+from pnpq.devices.odl_thorlabs_kbd101 import OdlThorlabs
+
+import pytest
+
+
+def test_disconnected_initialization():
+    with pytest.raises(RuntimeError):
+        OdlThorlabs("ABC", "DEF")
```

Comment:
Based on type of device in Thorlabs, the serial number starts with a specific number. It might be useful to check it in the unit test. For example for KB101, the serial number starts with '28' and with this format: '28xxxxxx'

Reply:
These values are intentionally invalid. I am testing failure to find a device.

The serial number format is useful information and that's good to know, but it's not relevant to us right now. In the future, we could parse the serial number and throw an exception if it doesn't start with a known value like `28`.